app/services/notifications.js

```diff
@@ -1,14 +1,11 @@
-import Ember from 'ember';
+import Service from 'ember-service';
+import get from 'ember-metal/get';
+import set from 'ember-metal/set';
 import {isAjaxError} from 'ember-ajax/errors';
-
-const {
-    Service,
-    computed: {filter},
-    A: emberA,
-    get,
-    set,
-    isArray
-} = Ember;
+import {isBlank} from 'ember-utils';
+import {A as emberA, isEmberArray} from 'ember-array/utils';
+import {filter} from 'ember-computed';
+import {dasherize} from 'ember-string';
```

Comment:
I started testing the move to modularisation through [ember-cli-shims](https://github.com/ember-cli/ember-cli-shims), working well so far, if it pans out it would be good to get a single PR put together that updates all of the imports.